packages/gatsby-theme-newrelic/src/components/SwiftSearch.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import SiteSearchAPIConnector from '@elastic/search-ui-site-search-connector';
+import {
+  SearchProvider,
+  WithSearch,
+  SearchBox,
+  Results,
+  Paging,
+  PagingInfo,
+} from '@elastic/react-search-ui';
+import ResultView from './ResultView';
+import PagingInfoView from './PagingInfoView';
+import SearchInput from './SearchInput';
+
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import PropTypes from 'prop-types';
+
+// TODO: move styles to emotion/wrapper
+import Icon from './Icon';
+import styles from '../styles/SwiftSearchStyles';
+
+const connector = new SiteSearchAPIConnector({
+  documentType: 'page',
+  engineKey: 'Ad9HfGjDw4GRkcmJjUut',
+});
+
+const configOptions = {
+  apiConnector: connector,
+  searchQuery: {
+    result_fields: {
+      title: {
+        snippet: {
+          size: 100,
+          fallback: true,
+        },
+      },
+      body: {
+        snippet: {
+          size: 400,
+          fallback: true,
+        },
+      },
+      url: {
+        raw: {},
+      },
+    },
+  },
+  initialState: {
+    resultsPerPage: 10,
+  },
+};
+
+const SwiftSearch = ({ className }) => {
```

Comment:
Nitpick: can we call this `SwiftTypeSearch` (or something with the full product name)? _Swift_ means something else and could get a little confusing.

Reply:
definitely, good point